app/scripts/models/account.js

```diff
@@ -605,9 +605,9 @@ define(function (require, exports, module) {
      * Check whether all the passed in permissions have been
      * seen previously.
      *
-     * @param {string} clientId
-     * @param {strings[]} permissions
-     * @returns {boolean} `true` if client has seen all the permissions,
+     * @param {String} clientId
+     * @param {Strings[]} permissions
```

Comment:
Oh, fair enough. Does it mean our trailing 's' on `Strings` is wrong in that case?